bigbluebutton-html5/imports/api/2.0/users/server/modifiers/kickUser.js

```diff
@@ -0,0 +1,39 @@
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Users from './../../';
+
+export default function kickUser(meetingId, userId) {
+  check(meetingId, String);
+  check(userId, String);
+
+  const selector = {
+    meetingId,
+    userId,
+  };
+
+  const modifier = {
+    $set: {
+      'user.kicked': true,
+      'user.connection_status': 'offline',
```

Comment:
these are not the user properties in 2.x api. Look here for example https://github.com/KDSBrowne/bigbluebutton/blob/93d7c9f4cea6dfa2ce6c65073400395f8a444ebc/bigbluebutton-html5/imports/api/2.0/users/server/modifiers/addUser.js#L78